### ISSUE TITLE
Add behavior to allow keyboard users to view a card's overflow text

### ISSUE THREAD

**Issue by clpetersonucf** [ucfopen/matching-materia-widget#65]
If a card in Matching has lengthy text, the overflow is visible on mouseover or to screen readers, but keyboard users cannot expand or select the tile to view the full text content.
